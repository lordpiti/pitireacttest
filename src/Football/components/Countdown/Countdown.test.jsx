import React from 'react';
import Countdown from './Countdown';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

describe('<Countdown /> with all data', () => {
  it(`should render correctly`, () => {
    //configure({ adapter: new Adapter() });
    const date = '2019-04-04';
    const metricsPanel = shallow(<Countdown date={date} />);
    expect(metricsPanel).toMatchSnapshot();
  });
});
