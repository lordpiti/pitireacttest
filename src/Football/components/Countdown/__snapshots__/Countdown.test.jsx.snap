// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Countdown /> with all data should render correctly 1`] = `
<div
  className="Countdown"
>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Days
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Hours
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Min
      </span>
    </span>
  </span>
  <span
    className="Countdown-col"
  >
    <span
      className="Countdown-col-element"
    >
      <strong>
        00
      </strong>
      <span>
        Sec
      </span>
    </span>
  </span>
</div>
`;
